pkg/system/phase2_creating.go

```diff
@@ -503,6 +507,15 @@ func (r *Reconciler) ReconcileGCPCredentials() error {
 	return err
 }
 
+// ReconcileIBMCredentials sets IsIBMCloud to indicate operator is running in IBM Cloud
+func (r *Reconciler) ReconcileIBMCredentials() error {
+	// Currently IBM Cloud is not supported by cloud credential operator
+	// In IBM Cloud, the COS Creds will be provided through Secret.
+	r.Logger.Info("Running in IBM Cloud")
+	r.IsIBMCloud = true
```

Comment:
Why aren't we loading the credentials secret here?

Reply:
You are right, we can actually load the credentials here instead of listing them in Phase4.
They should be ready here since we do not need to create cloud credentials CR and wait for it to reconcile.